<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite name="Failed suite [Surefire suite]" verbose="0">
  <test thread-count="5" name="Surefire test(failed)" verbose="0">
    <classes>
      <class name="MannyCoTest.Tests.SubmitOrderTests">
        <methods>
          <include name="exit"/>
          <include name="OrderHistoryValidation"/>
          <include name="submitOrder" invocation-numbers="0 1"/>
          <include name="launchApp"/>
        </methods>
      </class> <!-- MannyCoTest.Tests.SubmitOrderTests -->
      <class name="MannyCoTest.Tests.ErrorTest">
        <methods>
          <include name="productErrorValid"/>
          <include name="exit"/>
          <include name="lognErrorValid"/>
          <include name="launchApp"/>
        </methods>
      </class> <!-- MannyCoTest.Tests.ErrorTest -->
    </classes>
  </test> <!-- Surefire test(failed) -->
</suite> <!-- Failed suite [Surefire suite] -->
